Multiple container packages are not properly merged for packaging "aem-analyse"

When using a dedicated analysis Maven module with packaging "aem-analyse" like described in https://github.com/adobe/aemanalyser-maven-plugin/tree/main/aemanalyser-maven-plugin#example-for-a-multi-project-setup the container packages don't get properly merged.

To reproduce just build https://github.com/kwin/aem-analyser-example/tree/multicontainer-with-deps and observe the following error with just one container package being referenced inside https://github.com/kwin/aem-analyser-example/blob/a4e3bc413aadb2e21175be06c4022a7f4325d057/multicontainer-analyse/pom.xml#L23

```
[ERROR] [api-regions-exportsimports] biz.netcentric.aem:core:0.0.1-SNAPSHOT: Bundle core:0.0.1-SNAPSHOT is importing package(s) [com.adobe.aem.guides.wknd.core.models, com.adobe.acs.commons.util] in start level 20 but no bundle is exporting these for that start level. (biz.netcentric.aem:all:0.0.1-SNAPSHOT)
```

This is a correct observation, but once you uncomment the container package for only wknd in https://github.com/kwin/aem-analyser-example/blob/multicontainer-with-deps/multicontainer-analyse/pom.xml#L28-L34 (but not ACS AEM Commons) the error vanishes.

This is incorrect, as the ACS AEM Commons dependency can still not be resolved.

--- a/aemanalyser-core/src/main/java/com/adobe/aem/analyser/AemPackageConverter.java
+++ b/aemanalyser-core/src/main/java/com/adobe/aem/analyser/AemPackageConverter.java
@@ -135,6 +135,9 @@ public class AemPackageConverter {
     public void convert(final Map<String, File> contentPackages) throws IOException, ConverterException {
         final Map<String, String> properties = new HashMap<>();
 
+        if (contentPackages.size() > 1 && artifactIdOverride != null) {
+            throw new IllegalArgumentException("'artifactIdOverride' is not supported when multiple packages are provided!");
+        }
         final AclManager aclManager = new DefaultAclManager(null, ConverterConstants.SYSTEM_USER_REL_PATH_DEFAULT);
         final DefaultFeaturesManager featuresManager = new DefaultFeaturesManager(
             false,

--- a/aemanalyser-maven-plugin/src/main/java/com/adobe/aem/analyser/mojos/AemAnalyseMojo.java
+++ b/aemanalyser-maven-plugin/src/main/java/com/adobe/aem/analyser/mojos/AemAnalyseMojo.java
@@ -149,7 +149,6 @@ public class AemAnalyseMojo extends AbstractAnalyseMojo {
      */
     void convertContentPackages(final List<String> additionalWarnings, final List<String> additionalErrors) throws MojoExecutionException {
         final AemPackageConverter converter = new AemPackageConverter();
-        converter.setArtifactIdOverride(new ArtifactId(project.getGroupId(), project.getArtifactId(), project.getVersion(), null, "slingosgifeature").toMvnId());
         converter.setConverterOutputDirectory(getConversionOutputDir());
         converter.setFeatureOutputDirectory(getGeneratedFeaturesDir());
     

